packages/wrangler/src/deploy/index.ts

```diff
@@ -271,17 +270,17 @@ async function deployWorker(args: DeployArgs) {
 		);
 	}
 
-	const { userConfigPath } = resolveWranglerConfigPath(args, {
-		useRedirect: true,
-	});
-	const projectRoot = userConfigPath && path.dirname(userConfigPath);
 	const config = readConfig(args, { useRedirect: true });
 	if (config.pages_build_output_dir) {
 		throw new UserError(
 			"It looks like you've run a Workers-specific command in a Pages project.\n" +
 				"For Pages, please run `wrangler pages deploy` instead."
 		);
 	}
+	// We use the `userConfigPath` to compute the root of a Pages project,
```

Comment:
This looks like a normal deploy, not Pages?

Reply:
Yes you are right. Thanks